spec/draft-barnes-cfrg-4q.md

```diff
@@ -64,22 +79,6 @@ informative:
          -
               ins: P. Longa
 
-    Ed25519:
```

Comment:
In this context, would the better reference be to the EdDSA spec?  

https://tools.ietf.org/html/draft-irtf-cfrg-eddsa-02